tests/integration/sharding/test_sharding.py

```diff
@@ -0,0 +1,225 @@
+import os
+import random
+import string
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from jina import Document
+from jina.executors.indexers import BaseIndexer
+from jina.flow import Flow
+
+random.seed(0)
+np.random.seed(0)
+
+
+def get_index_flow(yaml_file, num_shards):
+    f = Flow().add(
+        uses='yaml/' + yaml_file,
```

Comment:
lets use cur_dir as in other tests to make this test runnable from different folders so that paths are absolute

Reply:
thanks, changed